### PR TITLE
Add Veridian & Transamerica

### PR DESCRIPTION
**Pull request by piquark6046**
Fix https://github.com/AdguardTeam/HttpsExclusions/issues/583

### CODE REVIEW

File: exclusions/banks.txt [hunk 1]
```diff
@@ -2684,6 +2685,7 @@ secure.upc.ua
 secure.wayforpay.com
 secure.ya.no
 secure.yakindogubank.com
+secure2.transamerica.com
```
Comment (zloyden) [inline exclusions/banks.txt:2688]: As I see, they also use `secure.transamerica.com`. I guess we can add it or just the whole domain.

Comment (Alex-302) [reply exclusions/banks.txt:2688]: Is it really a bank or similar service?

Comment (Alex-302) [reply exclusions/banks.txt:2688]: @piquark6046 Let's apply to `transamerica.com`.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: exclusions/banks.txt [hunk 2]
```diff
@@ -2048,6 +2048,7 @@ paragonbank.co.uk
 param.com.tr
 paranabanco.com.br
 paranabanco.com.br
+participant.transamerica.com
```
Comment (zloyden) [inline exclusions/banks.txt:2051]: I assume you can remove it safely since we have `transamerica.com`.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: exclusions/banks.txt (hunk 2 above)
Comment (Alex-302) [inline exclusions/banks.txt:2051]:
```suggestion
```

**Review by zloyden — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**